.tekton/tasks/echo-env.yaml

```diff
@@ -17,10 +20,42 @@ spec:
       script: |
         #!/bin/bash
 
-        echo "Node: $(node --version)"
+        if [ -n "$(params.prerelease)" ]; then
```

Comment:
No need to install version in this task.

Reply:
Yes only need npm-install and other tasks. I will update